### DISCUSSION TITLE
Table: Column filter add "All" option after search for a value

### DISCUSSION THREAD

**Discussion by liner-lin** [grafana/grafana#38126] [Dashboards, Panels & Visualizations]
![image](https://user-images.githubusercontent.com/25217954/124536407-8a0d3680-de4a-11eb-858a-b5dd795a64ec.png)

**What would you like to be added**:

**Why is this needed**:
I want to select all filter values.

**Comment by torkelo** [comment]
that does not make much sense, you men clear all filters?

**Comment by torkelo** [comment]
> I want to select all filter values.

this would be the same as no values selected , and have no impact on the table (no filtering)

**Comment by liner-lin** [comment]
> > I want to select all filter values.
> 
> this would be the same as no values selected , and have no impact on the table (no filtering)

I want to select all filtered values. Just like the excel filter function, there is an option to select all search results.
![image](https://user-images.githubusercontent.com/25217954/124547716-22adb180-de5f-11eb-9f6e-00e731817fc3.png)

**Comment by torkelo** [comment]
ah, after a search is entered, then it makes sense

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by scarabus** [comment]
This would be a useful feature. Also the option to have case-insensitive searching would be useful too.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
